tests/integration/onboarding.selectors.ts

```diff
@@ -1,11 +1,12 @@
 export enum OnboardingSelectors {
   AnalyticsAllowBtn = 'analytics-allow-btn',
+  AnalyticsDenyBtn = 'analytics-deny-btn',
   BackUpSecretKeyBtn = 'back-up-secret-key-btn',
   NewPasswordInput = 'set-or-enter-password-input',
   ConfirmPasswordInput = 'confirm-password-input',
   SetPasswordBtn = 'set-password-btn',
   SignInLink = 'sign-in-link',
   SignInBtn = 'sign-in-btn',
-  SignUpBtn = 'sign-up-btn',
+  SignUpBtn = 'sign-up',
```

Comment:
Why is this being removed?

Reply:
This is for the sign up button in the vercel app this matches the element there

Reply:
This is used in the `demo.page.ts`, so the selector should just be used there and kept as `sign-up-btn`. The point of using constants is to make sure the string is accurate in all places. There is no point to use a constant in one place and then type out the string again in another.